src/select.scss

```diff
@@ -42,13 +42,34 @@ $block: #{$fd-namespace}-select;
       }
     }
 
-    @include fd-fiori-focus();
     @include fd-form-states();
 
+    @include fd-focus() {
+      outline: 0;
+
+      &::before {
+        content: '';
+        outline-offset: -0.1875rem;
+        outline-width: var(--sapContent_FocusWidth);
+        outline-color: var(--sapContent_FocusColor);
+        outline-style: var(--sapContent_FocusStyle);
+        height: 100%;
+        width: 100%;
+        z-index: 1;
+        right: 0.0234375rem;
+        position: absolute;
```

Comment:
Maybe it will be better to have left/top/right/bottom as 0.1875rem and border instead of outline?

Reply:
changed to border